src/test_l2_hdmi_cec_driver.c

```diff
@@ -1,77 +1,394 @@
+        return;
+    }
+
+    // Step 2: Call the API HdmiCecGetPhysicalAddress() with the handle obtained from HdmiCecOpen() and a pointer to an unsigned integer to store the physical address.
+    status = HdmiCecGetPhysicalAddress(handle, &physicalAddress);
+    UT_LOG("Invoking HdmiCecGetPhysicalAddress with handle: %d", handle);
+
+    // Step 3: Check the return status of HdmiCecGetPhysicalAddress(). It should return HDMI_CEC_IO_SUCCESS for a valid handle and physical address.
+    UT_ASSERT_EQUAL(status, HDMI_CEC_IO_SUCCESS);
+    if(status != HDMI_CEC_IO_SUCCESS)
+    {
+        UT_LOG("HdmiCecGetPhysicalAddress failed with status: %d", status);
+        // Call the post-requisite API HdmiCecClose() with the handle obtained from HdmiCecOpen().
+        HdmiCecClose(handle);
+        return;
+    }
+
+    // Step 4: Validate the physical address obtained. It should be in the range of 0 to (((0x04 &0xF0 ) << 20)|( (0x04 &0x0F ) << 16) |((0x04 & 0xF0) << 4)  | (0x04 & 0x0F)).
+    unsigned int maxAddress = (((0x04 &0xF0 ) << 20)|( (0x04 &0x0F ) << 16) |((0x04 & 0xF0) << 4)  | (0x04 & 0x0F));
+    UT_ASSERT_TRUE(physicalAddress >= 0 && physicalAddress <= maxAddress);
+    if(!(physicalAddress >= 0 && physicalAddress <= maxAddress))
+    {
+        UT_LOG("Physical address: %u is out of range", physicalAddress);
+        // Call the post-requisite API HdmiCecClose() with the handle obtained from HdmiCecOpen().
+        HdmiCecClose(handle);
+        return;
+    }
+
+    // Step 5: Call the post-requisite API HdmiCecClose() with the handle obtained from HdmiCecOpen().
+    status = HdmiCecClose(handle);
+    UT_ASSERT_EQUAL(status, HDMI_CEC_IO_SUCCESS);
+    if(status != HDMI_CEC_IO_SUCCESS)
+    {
+        UT_LOG("HdmiCecClose failed with status: %d", status);
+    }
+
+    UT_LOG("Exiting test_l2_hdmi_cec_driver_GetPhysicalAddress...");
+}
+
+/**
+* @brief Test the transmission of CEC command in HDMI CEC driver
+*
+* This test case tests the transmission of CEC command in HDMI CEC driver. It checks if the CEC command is transmitted successfully and the correct status is returned. This is important to ensure the HDMI CEC driver is working as expected.
+*
+* **Test Group ID:** 02@n
+* **Test Case ID:** 006@n
+*
+* **Test Procedure:**
+* Refer to UT specification documentation [l2_hdmi_cec_driver_test_specification.md](l2_hdmi_cec_driver_test_specification.md)
+*/
+
+void test_l2_hdmi_cec_driver_TransmitCECCommand(void)
+{
+    UT_LOG("Entering test_l2_hdmi_cec_driver_TransmitCECCommand...");
+
+    int handle;
+    HDMI_CEC_STATUS status;
+    unsigned char cecPacket[16] = {0x0F, 0x9E, 0x00}; // CEC Version command to non-existent logical address
+    int len = 3;
+    int result;
+
+    // Step 1: Call HdmiCecOpen
+    status = HdmiCecOpen(&handle);
+    UT_LOG("Invoking HdmiCecOpen(). Status: %d", status);
+    UT_ASSERT_EQUAL(status, HDMI_CEC_IO_SUCCESS);
+
+    // Step 2: Prepare a CEC packet is already done
```

Comment:
Before transmit, we should add the logical address (for the DUT) and then use a the destination address some thing else (as we don't have any connection here, any address should be fine.  But please be noted that there is some change coming for this test scenario.

Reply:
HdmiCecAddLogicalAddress() is not available in APi list of test requirement

Reply:
With the updated test requirement , test scenario worked as expected.